docs/source/contribution/technical_steering_committee.md

```diff
@@ -36,18 +39,38 @@ On this page we describe:
 ## Requirements to become a maintainer
 
 Just contributing does not make you a maintainer; you need to demonstrate commitment to Kedro's long-term success by
-working with existing maintainers for a period of time.
+working with existing maintainers for at least one year.
 
-We look for commitment markers who can do the following:
+We look for commitment markers who can do at least some of the following:
 
 - Write high-quality code and collaborate with the team and community
+- Make substantial improvements to our documentation, either in terms of content or information architecture
+- Bring awareness of industry trends to the team, connect the project to complementary initiatives, help steer the direction
```

Comment:
Maintainer responsibility

Maybe reword to something like "Steer the direction of the project to ensure it retains relevance, including by bringing awareness of industry trends to the team and connecting the project to complementary initiatives"

(Don't have to take my wording, just trying to get my point across)